### ISSUE TITLE
Fix Tests for Debian 10 & Ubuntu 20.04 (Cassandra Restart)

### ISSUE THREAD

**Issue by dallinb** [locp/ansible-role-cassandra#85]
**Describe the bug**
At the moment, we set `cassandra_service_restart` to false for Debian 10 and Ubuntu 20.04 because we have problems with the service restart during the test converge in Molecule.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go with the default value for `cassandra_service_restart` and run the Molecule tests.  It fails for the two mentioned distros.

**Expected behaviour**
All supported distros should be able to restart Cassandra if required.

**Comment by dallinb** [comment]
Looking on Debian 10, the service goes into the following state:

```shell
root@debian10:~# sysctl status cassandra
sysctl: cannot stat /proc/sys/status: No such file or directory
sysctl: cannot stat /proc/sys/cassandra: No such file or directory
root@debian10:~# service cassandra status
[FAIL] could not access pidfile for Cassandra ... failed!
```

But Cassandra is still running:

```shell
root@debian10:~# nodetool status
Datacenter: DC1
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Owns (effective)  Host ID                               Token                                    Rack
UN  172.17.0.3  65.29 KiB  100.0%            0c2abd28-b2ce-49a5-b05c-54c19b58707e  -6399235422545095697                     RACK1
```

**Comment by dallinb** [comment]
Looks like this may have been fixed in Cassandra 3.11.7 as part of [CASSANDRA-15770](https://issues.apache.org/jira/browse/CASSANDRA-15770).  We're currently testing against 3.11.6 as that is what is available in the package repo at the moment.
